helm/falco/templates/podlogs.yaml

```diff
@@ -0,0 +1,162 @@
+{{- if .Values.observability.enabled }}
+---
+# PodLogs for falco-exporter (all logs - non-sensitive metrics only)
+apiVersion: monitoring.grafana.com/v1alpha2
+kind: PodLogs
+metadata:
+  name: falco-exporter-logs
+  namespace: {{ .Release.Namespace }}
+  labels:
+    giantswarm.io/managed-by: security-team
+    giantswarm.io/service-type: managed
```

Comment:
`managed-by` probably is irrelevant since we have a `team` label, but `service-type` we have a value for it I believe